images/base/Dockerfile.centos7

```diff
@@ -4,9 +4,9 @@
 #
```

Comment:
Why does `images/base/Dockerfile` not work for you? What is different about this `.centos7` version that you need?